src/v2/Apps/Fair/FairApp.tsx

```diff
@@ -140,9 +142,18 @@ const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
 
 const TrackingWrappedFairApp: React.FC<FairAppProps> = props => {
   const {
-    fair: { internalID },
+    fair: { internalID, profile },
   } = props
+
   const { contextPageOwnerSlug, contextPageOwnerType } = useAnalyticsContext()
+
+  // If a fair's profile is inaccessible, that means it's private, which in turn means
```

Comment:
Do we handle the `published` and `private` fields in gravity or metaphysics?

Reply:
Gravity: one gets a 403 if you request a `private` profile.